README.md

```diff
@@ -27,22 +27,20 @@ values in [kube/](kube/) example files.
 * `NODE_MASTER`: Whether this node can be a master node. Default: `true`.
 * `NODE_DATA`: Whether this node can be a data node. Default: `true`.
 * `NODE_INGEST`: Whether this node can be a data ingesting node. Default: `true`.
-* `HTTP_ENABLE`: Whether this node can be a client (HTTP) node. Default: `true`.
 * `HTTP_BIND_HOST`: http bind address.. Default: `0.0.0.0`.
-* `KUBERNETES_SERVICE`: kubernetes service name for master nodes. Default `elasticsearch-master`.
-* `ENABLE_TRANSPORT_SSL`: whether to enable search-guard transport SSL. Default: `false`.
+* `THREAD_POOL_QUEUE_SIZE` - see https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-threadpool.html
```

Comment:
Can we rename it to `THREAD_POOL_WRITE_QUEUE_SIZE` so that it's clear which thread pool this is for specifically?